### ISSUE TITLE
Remove Gutenberg rollout logic

### ISSUE THREAD

**Issue by guarani** [wordpress-mobile/gutenberg-mobile#2901]
The Gutenberg rollout in the WordPress apps, which opted-in users to the new editor, is complete.

After disabling the block editor informative popup (https://github.com/wordpress-mobile/WordPress-iOS/pull/15492 and https://github.com/wordpress-mobile/WordPress-Android/pull/13593), we need to remove the associated logic that showed the popup, as well as the rest of the Gutenberg rollout code on both platforms.

**Comment by hypest** [comment]
Yes, let's remove this logic when we'll work on removing the editor switch from the settings.

**Comment by mchowning** [comment]
:wave: @hypest ! So we want to remove the logic that checks the backend for the default mobile-editor and, instead, the app will now always use Gutenberg as the default mobile editor? I believe this means that the Aztec editor will only be used when opening a post that was edited with the classic editor on the web. Related to this, should we continue to update the mobile-editor default on the backend, or drop that as well?

**Comment by hypest** [comment]
👋 @mchowning, I think the scope of this ticket is about the logic for migrating the default to Gutenberg from Classic/Aztec. That's the part we should cleanup. The remaining logic that governs which editor is shown and when will need to be consolidated with all the various cases (for example, whether the site has Classic as default on the web), and is part of the mini-project we'll embark soon ("removing the editor switch from the settings"). Does that answer your question though? Let me know.

**Comment by mchowning** [comment]
> The remaining logic that governs which editor is shown and when will need to be consolidated with all the various cases (for example, whether the site has Classic as default on the web), and is part of the mini-project we'll embark soon ("removing the editor switch from the settings").

That makes sense @hypest . I was just trying to see if we had a clear goal on this such that it might be something I could start doing a bit of work on. It sounds like perhaps we haven't clearly defined the desired behavior yet though?

**Comment by hypest** [comment]
Removing some logic now is of lower priority and it's best to just work on this as part of the "remove the editor switch" project, that's pretty much it 😄

**Comment by guarani** [comment]
I've updated the issue title and description to include removing all Gutenberg rollout code, since that process is over.
